nitransforms/io/afni.py

```diff
@@ -18,6 +19,39 @@ def __str__(self):
         param = self.structarr['parameters']
         return '\t'.join(['%g' % p for p in param[:3, :].reshape(-1)])
 
+    def to_ras(self, moving=None, reference=None):
+        """Convert to RAS+ coordinate system."""
+        pre = LPS.copy()
```

Comment:
just to simplify initialization

```suggestion
        pre = post = LPS.copy()
```